peer/voting/voting_test.go

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) 2014-2019 Bitmark Inc.
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package voting
```

Comment:
I dont think put test file in the same package of source file is a good solution. In my opinion, the best practice when writing unit test is put the test file and source file in different package, it help you to test public function like a real user. The internal/private function is not necessary to test, we need to test the caller instead. You can consider all test files in your PR.

Reply:
it is encusplation of object

Reply:
Don't understand your idea. :|